app/test/check_tx_test.go

```diff
@@ -44,6 +45,7 @@ func TestCheckTx(t *testing.T) {
 		checkType        abci.CheckTxType
 		getTx            func() []byte
 		expectedABCICode uint32
+		expectedLog      string
```

Comment:
Does the expected log really help if we now have error codes we can assert

Reply:
i guess it's redundant in check tx